### PR TITLE
[DSS] DDP-7714: Add tabular block 

### PR DESCRIPTION
**Pull request by alextanlan**
Implemented a new tabular block (see related [ticket DDP-7714](https://broadinstitute.atlassian.net/browse/DDP-7714)).
    Eventually, we agreed to have a tabular block content as a flat (one-dimensional) list of question blocks. Every of the question blocks has its own `columnSpan` property. If a question has a prompt the question `columnSpan` should should be increased by one (the question title/prompt should span a separate cell)

--------------
**_Update July, 16:_**
**_Real data example (basil dev)_**
res1

--------------

**_Screenshots (with mock data):_**

**_Notes:_**
As for additional boolean question (switcher of enable/disable state of some composite child questions, see `Not applicable` checkbox on the image below) - we can keep it as a last child of the composite question and customize styles specifically for that particular block/question. 
image

**_Example with custom styles (with mock data):_**
Untitled2

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.tabular-container.composite {
+    border: none;
+    width: 70%;
+
+    * {
+        border: none;
+    }
+
+    .questions--row {
```
Comment (josemanuel167) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.scss]: is this double dash intended?

**Review by josemanuel167 — DISMISSED** [review]
LGTM

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.scss (hunk 1 above)
Comment (alextanlan) [reply ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/tabularBlock/tabularBlock.component.scss]: Actually not. 
There was no any special sense there. 
Fixed.

**Review by josemanuel167 — DISMISSED** [review]
LGTM x2, pending comments for discrepancy on numberOfColumns

**Review by josemanuel167 — DISMISSED** [review]

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**